tests/suite/custom_resources_utils.py

```diff
@@ -329,14 +329,15 @@ def patch_virtual_server_from_yaml(
     :return:
     """
     print(f"Update a VirtualServer: {name}")
-    with open(yaml_manifest) as f:
-        dep = yaml.safe_load(f)
 
-    try:
-        custom_objects.patch_namespaced_custom_object(
-            "k8s.nginx.org", "v1", namespace, "virtualservers", name, dep
+    try: 
+        delete_virtual_server(
+            custom_objects, name, namespace
+        )
+        create_virtual_server_from_yaml(
+            custom_objects, yaml_manifest, namespace
         )
-        print(f"VirtualServer updated with name '{dep['metadata']['name']}'")
```

Comment:
was this removal intentional?

Reply:
Fixed in https://github.com/nginxinc/kubernetes-ingress/pull/1230/commits/d268b7c2fb71e4037891035e2c2ac867ef9a419c